manual/src/threelang/tfgrid/funkwhale.md

```diff
@@ -0,0 +1,34 @@
+# Funkwhale Namespace
+
+- To deploy a funkwhale instance, use the funkwhale namespace.
+
+## Create Operation
+
+- action name: !!tfgrid.funkwhale.create
```

Comment:
Can you add an example for every action? I mean a real life example. For example:
```
!!tfgrid.funkwhale.create
       name: myfunkwhale
       farm_id: 251
       capacity: medium
       ssh_key: mysshkey
       admin_mail: brandon@mycompany.com
       admin_username: brandon
       admin_password: mLKJLKJajmdslkqjd
```